extensions/wikia/PhalanxII/fallback/Phalanx.class.php

```diff
@@ -450,41 +450,41 @@ static function findBlocked($text, $blocksData, $writeStats = true, &$matchingBl
 					);
 				}
 				wfRestoreWarnings();
-				if ($matched) {
-					$blockData = self::getFromId($blockId);
+				if ( $matched ) {
+					$blockData = self::getFromId( $blockId );
 					if ( $blockData ) {
-						if ($writeStats) {
-							self::addStats($blockData['id'], $blockData['type']);
+						if ( $writeStats ) {
+							self::addStats( $blockData['id'], $blockData['type'] );
 						}
 						$result['blocked'] = true;
 						$result['msg'] = $matches[0];
 					}
 				}
-			} else { //plain text
-				if (!$isCase) {
-					$text = strtolower($text);
-					$blockText = strtolower($blockText);
+			} else { // plain text
+				if ( !$isCase ) {
+					$text = strtolower( $text );
+					$blockText = strtolower( $blockText );
 				}
-				if ($isExact) {
-					if ($text == $blockText) {
-						$blockData = self::getFromId($blockId);
+				if ( $isExact ) {
+					if ( $text == $blockText ) {
+						$blockData = self::getFromId( $blockId );
 						if ( $blockData ) {
-							if ($writeStats) {
-								self::addStats($blockData['id'], $blockData['type']);
+							if ( $writeStats ) {
+								self::addStats( $blockData['id'], $blockData['type'] );
 							}
 							$result['blocked'] = true;
-							$result['msg'] = $origText;    //original case
+							$result['msg'] = $origText;    // original case
 						}
 					}
 				} else {
-					if ( !empty($blockText) && strpos($text, $blockText) !== false) {
-						$blockData = self::getFromId($blockId);
+					if ( !empty( $blockText ) && strpos( $text, $blockText ) !== false ) {
+						$blockData = self::getFromId( $blockId );
 						if ( $blockData ) {
-							if ($writeStats) {
-								self::addStats($blockData['id'], $blockData['type']);
+							if ( $writeStats ) {
+								self::addStats( $blockData['id'], $blockData['type'] );
 							}
 							$result['blocked'] = true;
-							$result['msg'] = $origText;    //original case
+							$result['msg'] = $origText;    // original case
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)